src/components/DpDataTable/DpDataTable.vue

```diff
@@ -514,6 +536,30 @@ export default {
       this.toggleSelectAll(false)
     },
 
+    setColsWidth (headers) {
+      Array.from(headers).forEach(tableHeaderEl => {
+        /**
+         * Some of childNodes of the first table row are not Element nodes but comments or text.
+         * This originates in the Vue template compiler leaving empty html comments when rendering
+         * falsy `v-if` blocks. We allow only nodeType "Element" to access its `getBoundingClientRect` api.
+         */
+        if (tableHeaderEl.nodeType === 1) {
+          const headerField = tableHeaderEl.getAttribute('data-col-field')
+          const savedColWidth = sessionStorage.getItem(`data-col-field=${headerField}`)
+          const fixedWidth = this.getFixedColWidth(headerField)  // some columns, such as 'flyout' and 'wrap', should not be resizable; their width is fixed; the getBoundingClientRect() function should not be applied to them
```

Comment:
Nitpick: this comment could as well live in a `/**` block above the function definition of `getFixedColWidth`.

Reply:
done here https://github.com/demos-europe/demosplan-ui/pull/835/commits/eda5a1c80fa38a01472b3b0964ee94db7da2fe9f